low-level/sspl_ll_d

```diff
@@ -715,6 +716,9 @@ def send_thread_controller_request(state):
 #     logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
 #     logger.setLevel(log_level)
 
+def initialize_iem():
+    EventMessage.init(component='sspl', source='S')
```

Comment:
again, capture errors. log.debug in case of success, log.error in case of error encountered

Reply:
Done